### ISSUE TITLE
Listening to a specific device

### ISSUE THREAD

**Issue by IamTheCarl** [lukaslueg/dumpcap-rust#2]
Wonderful library. It works great so far but I'd like to select which device I'm listening to.
It currently just selects the first device on my system and listens to only that one. (in this case the wifi)

How would I set it to a different one? (I want it to listen to the Ethernet)

**Comment by lukaslueg** [comment]
Uhm haven't had a look for years on this. `Argument::default().device_argument("eth1").build()` should give you arguments to capture on `eth1`.

**Comment by IamTheCarl** [comment]
I saw that and it looked like it should serve the purpose.
The only issue is that it's private.
If it was meant to be public then I'll just make a modification.

**Comment by lukaslueg** [comment]
Yeah, although I have to say, I really can't remember what's what here :-/ If you like, you could also update the integration test
